src/app/admin/deleteUser/DeleteUserDialog.tsx

```diff
@@ -36,9 +36,14 @@ const DeleteUserDialog: React.FC<Props> = (props) => {
             return;
         }
 
-        const { error } = await adminDeleteUser(props.userToDelete.id);
+        const { error: deleteUserError } = await adminDeleteUser(props.userToDelete.id);
 
-        if (!error) {
+        if (deleteUserError) {
+            props.setAlertOptions({
+                success: false,
+                message: <>{`${deleteUserError.type}. Log ID: ${deleteUserError.logId}`}</>,
```

Comment:
since `deleteUserError.type` is meant to be just an identifier of the error, we should compute the user friendly error message based on that and display it there :)